modules/sdk-coin-atom/.mocharc.yml

```diff
@@ -1,5 +1,5 @@
 require: 'ts-node/register'
-timeout: '60000'
+timeout: '180000'
```

Comment:
Initially the local test suite was timing out around challenge generation (which took somewhere around 2 - 3 mins); To avoid that, I had bumped the limit but now that we're mocking the challenge generation part, we wouldn't have to increase this limit. Made the change to bring it back to default